src/app/features/rpc-transaction-request/transaction-account-signer/transaction-account-signer.tsx

```diff
@@ -10,20 +10,18 @@ import { useCurrentAccountIndex } from '@app/store/accounts/account';
 import { useStacksAccounts } from '@app/store/accounts/blockchain/stacks/stacks-account.hooks';
 import { AccountAvatarItem } from '@app/ui/components/account/account-avatar/account-avatar-item';
 
-interface SwitchAccountTriggerProps {
+interface TransactionAccountSignerProps {
   address: React.ReactNode;
   availableBalance: Money;
   fiatBalance: Money;
   isLoadingBalance: boolean;
-  onSwitchAccount(): void;
 }
-export function SwitchAccountTrigger({
+export function TransactionAccountSigner({
```

Comment:
Only feedback is on this choice of name. Naming it with "signer" implies it does something, whereas we're just using it to display the account, as I understand? 

`SigningAccountCard`
`TransactionToBeSignedBy`
`SigningAccountDisplayer`

not 100% on any of these but some ideas

Reply:
Yeah, good point, will change it.